change/@internal-react-composites-6e1eac11-db67-47a8-85f8-1d0788a935a8.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "[BREAKING CHANGE] Update createAzureCommunicationMeetingAdapter arguments to enable accepting just a teams link without having to provide an extracted chat thread ID.",
```

Comment:
Why is this a breaking change if you are changing a beta API?

Reply:
I understand this would break the beta usage of MeetingComposite. Is that what you meant?

Reply:
Yeah, I wanted to highlight in the beta notes in the changelog this is a very jarring change (customers using meeting composite _will_ be broken)